Skuid_Techniques/Navigation/README.MD

```diff
@@ -0,0 +1,167 @@
+  
+### Display data in the navigation element 
+Because the iterator components are bound to model data - it is easier to use data.  You can also add HTML to templates in the list. 
+
+You can also use style variant conditions to highlight style of a wrapper in a deck card if certain conditions are met.  Alert to “urgent” attention needed on one navigation element. 
+
+### Disable navigation element based on data.  
+We do not have Enable Condition logic for the interactions associated with Deck Cards or List items.  (Enhancement!)   Instead use Branch Actions to prevent the action for particular scenarios.  You have to define the actions for each navigation item,  but this allows you to have different logic rules for each item.  Your failure branch can post a message about why the page load is not happening. 
+
+### Collapsable Nav 
+Just like the vertical navigation component  - the most straightforward method to have collapsibility is to have 2 components and use conditional rendering to only show one. 
+
+The list component needs to implement column based conditional rendering.  (Enhancement!) 
+
+A deck component will not work so well here.  The collapsed state will want to wrap individual cards instead of shrinking the entire column.   This is a work in progress. 
+ 
+### Secondary navigation 
+The list component allows drawers.  You can configure the drawers to open on click or with the secondary drawer icon.   You can put a second list in the drawer and use context conditions to ensure that the correct sub nav items show up under their parents. 
+
+A current limitation is that the selected state of child lists do not interact with the selected state of the parent list.   Also - it would be really nice to have “close all drawers” action so that when one parent item is selected - it first closes all the others.  
+
+With a Deck - a secondary deck can be conditionally rendered only to show when the “selected” flag on a navigation element is true.  A branch on the interaction can set the value back to false if it is already true - and in that way reproduce the open / close toggle of the navigation component or list drawers. 
+
+### Remember my place. 
+Here the solution is not as straightforward as the navigation component. The list will not listen to URL Hashes.  If you are already doing some server side work to track progress - leverage that data to open the users view in the right place. On page load, have some separate model that loads the correct page include value and uses a style variant condition to make the correct row of the navigation look selected. 
+
+### Data Driven navigation. 
+With a list - you can retrieve model data to drive exactly what will show on the page.  This would allow you to control the navigation by user, or by profile, or by progress made. 
```

Comment:
```suggestion
With a List - you can retrieve model data to drive exactly what will show on the page.  This would allow you to control the navigation by user, or by profile, or by progress made. 
```